pandas/_libs/hashtable_func_helper.pxi.in

```diff
@@ -259,7 +259,7 @@ def ismember_{{dtype}}({{scalar}}[:] arr, {{scalar}}[:] values):
     {{if dtype == 'object'}}
     for i in range(n):
         val = arr[i]
-        k = kh_get_{{ttype}}(table, <PyObject*> val)
+        k = kh_get_{{ttype}}(table, <PyObject*>val)
```

Comment:
are actually linting this? iirc these were changed to add the space last year. i don’t care which way but we should have 1 true way

Reply:
I'm not aware of a way to lint for this.

I've been pretty consistent about no-space, so at this point that constitutes a pretty large majority of existing cases.